index.html

```diff
@@ -273,35 +273,49 @@ <h4 class="connect">LET'S CONNECT</h4>
             </div>
     
         </section>
-
-        <section id="contact" class="form-container">
-            <div class="form-min">
-                <div class="form-des">
-                    <h1>Contact me</h1>
-                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-                </div>
-
-               <form id="form" method="POST" action="https://formspree.io/f/xjvzplnw">
-                    <ul class="form-ul">
-                        <li>
-                            <input type="text" maxlength="30" name="name" id="user-name" placeholder="Enter your name here" required>
-                        </li>
-                        <li>
-                            <input type="email" name="email" id="email-address" placeholder="Enter your email here" required>
-                        </li>
-                        <li>
-                            <textarea name="message" id="msg" maxlength="500" placeholder="Write your message here " required></textarea>
-                        </li>
-                        <span id="form-error" class="contact-top-p"></span>
-                        <li class="btn">
-                            
-                            <button type="submit" class="form-button">Get In Touch</button>
-                        </li>
-                    </ul>
-                </form>
-    
+        
+        <section class="contact">
+            <div class="descr">
+              <h1>Contact me</h1>
+              <p>
+                If you have an application you are interested in developing, a feature
+                that you need built or a project that needs coding. I’d love to help
+                with it!
+              </p>
+              <form action="https://formspree.io/f/mdovljrd" method="POST">
+                <ul class="the-relative">
+                  <li class="tname">
+                    <input
+                      type="text"
+                      name="username"
+                      maxlength="30"
+                      placeholder="Your name"
+                      required
+                    />
+                  </li>
+                  <li class="mail">
+                    <input
+                      type="email"
+                      name="email"
+                      placeholder="Your email"
+                      required
+                    />
+                  </li>
+                  <li class="texts">
+                    <input
+                      type="text"
+                      name="Message"
+                      maxlength="500"
+                      class="text-area"
+                      placeholder="Write your message here..."
+                      required
+                    />
+                  </li>
+                </ul>
+                <button class="submit">Get in touch</button>
```

Comment:
[OPTIONAL] Please stick to the Figma design your button all effects are not the same as the Figma design kindly fix your all button effects :+1: 

![Screenshot from 2023-02-17 17-31-04](https://user-images.githubusercontent.com/45226848/230716383-2465d2c1-d7a2-4520-893c-ab6551eec0f3.png)

Reply:
thank you so much i will go through you advice